fondat/aws/cloudwatch.py

```diff
@@ -0,0 +1,58 @@
+import aiobotocore
+import fondat.codec
+import fondat.pagination
+import logging
+
+from collections.abc import Iterable
+from fondat.aws import Client
+from fondat.codec import Binary, String
+from fondat.error import InternalServerError, NotFoundError
+from fondat.resource import resource, operation
+from fondat.security import SecurityRequirement
+from typing import Any
+from fondat.monitoring import Measurement, Counter, Gauge, Absolute
+
+
+_logger = logging.getLogger(__name__)
+
+
+def cloudwatch_resource(
+    *,
+    client: Client,
+    security: Iterable[SecurityRequirement] = None,
+):
+
+    if client.service_name != "cloudwatch":
+        raise TypeError("expecting cloudwatch client")
+
+    @resource
+    class Metric:
+        @operation(security=security)
+        async def put_metric(self, measurement: Measurement):
```

Comment:
1. Operations must be named one of: `get`, `put`, `post`, `delete`, `patch`. I should probably enforce this in fondat-core. One option is to use `@mutation` instead of `@operation`, which I can explain tomorrow. But I see you want to post alerts as well, so this has me wonder if there should just be a `Metrics` resource and an `Alerts` resource, each with a `post` method?
2. I see you're using the `Measurement` object, which should be fine for now. I wonder though if we'll ever want to post other metric data?